projects/plugins/jetpack/modules/woocommerce-analytics/classes/class-jetpack-woocommerce-analytics-universal.php

```diff
@@ -277,6 +362,28 @@ public function checkout_process() {
 	public function order_process( $order_id ) {
 		$order = wc_get_order( $order_id );
 
+		$payment_option = $order->get_payment_method();
+
+		if ( is_object( WC()->session ) ) {
+			$create_account = true === WC()->session->get( 'wc_checkout_createaccount_used' ) ? 'Y' : 'N';
+		} else {
+			$create_account = 'N';
+		}
+
+		$guest_checkout = $order->get_user() ? 'N' : 'Y';
+
+		$express_checkout = 'null';
+		// When the payment option is woocommerce_payment
+		// See if Google Pay or Apple Pay was used.
+		if ( 'woocommerce_payments' === $payment_option ) {
```

Comment:
This block of code is tightly coupled to WooCommerce payments. Perhaps it could be factored out into a function so it's clear it depends on another plugin. 

Or perhaps use a hook/filter to provide a standard interface to any plugin that might need to supply this info (e.g. other gateway plugins).

Also see above comment – if there's a Woo API for express payment methods, we should use that. If there isn't an API - should we add one?